.gitignore

```diff
@@ -52,6 +52,7 @@ flymake*
 # mkdocs
 docs/latest
 docs/latest.yaml
+docs/cspell.json
```

Comment:
I'm not really familiar with cspell—is the config file always named "cspell.json"? Would it always need to be at the root of the `docs` directory?

Reply:
Tagging @stevenGravy to answer, as I just stole his config to start using for myself without knowing what's standard.